nestle.py

```diff
@@ -583,6 +651,13 @@ def update(self, pointvol):
         self.ell = bounding_ellipsoid(self.points, pointvol=pointvol)
         self.ell.scale_to_vol(pointvol)
 
+    def propose_point(self, u):
+        while True:
+            new_u = u + scale * self.ell.randoffset(rstate=self.rstate)
```

Comment:
`scale` hasn't been passed to this function from `new_point`.

I think the original behavior is actually not correct: one shouldn't change the proposal distribution during the MCMC: a constant scale should be used during the entirety of `new_point`. I think it is fine to change the proposal distribution (using `scale`) between calls to `new_point` though. So perhaps `scale` should become an attribute of the class that is updated at the end of each `new_point` call.